### ISSUE TITLE
Colocar logos da transparência brasil e do IFAL nos relatórios

### ISSUE THREAD

**Issue by danielfireman** [dadosjusbr/analises#7]
IFAL - https://www2.ifal.edu.br/o-ifal/comunicacao/arquivos/logos/copy_of_IFALvertical.png

@nazareno .. também precisamos do logo da UFCG, certo?

**Comment by nazareno** [comment]
Sim. Inclusive nem precisa do analytics. Basta UFCG, IFAL e TB. Agora, preferimos assim ou só dadosjubr e colocamos os realizadores em outro lugar, tipo no site e/ou readmes, @danielfireman ?

**Comment by danielfireman** [comment]
Bom ponto, @nazareno. Acho que só o dadosjusbr talvez esteja bom.

**Comment by rdurl0** [comment]
O logo da TB (200x200px):

![TBrasil_200](https://user-images.githubusercontent.com/22898231/109334103-078ebd80-783f-11eb-9f03-ddde362cc70b.png)

Ou na versão horizontal (768x212):
![TB-logo-Horizontal-768x212](https://user-images.githubusercontent.com/22898231/109335136-64d73e80-7840-11eb-831e-78af49d54af1.png)

**Comment by danielfireman** [comment]
> Sim. Inclusive nem precisa do analytics. Basta UFCG, IFAL e TB. Agora, preferimos assim ou só dadosjubr e colocamos os realizadores em outro lugar, tipo no site e/ou readmes, @danielfireman ?

Não tenho preferência, Naza. Mas fiquem a vontade para  colocar nos READMEs dos repositórios principais (podem mandar PRs).

Valeu, @rdurl0 pelo logo. Já estou em contato com Mari para colocar no site.
